src/bluesearch/widgets/article_saver.py

```diff
@@ -251,6 +250,7 @@ def pdf_report(self, output_dir=None):
 
         Returns
         -------
+        output_file_path : str
```

Comment:
Was this line duplicated by accident? Also I just noticed that the return type is actually `pathlib.Path` and not `str`.

Reply:
Thanks! Fixed in 0d9b563dd67d7b79201fae60f90394d549a2369a.